aea/cli/scaffold.py

```diff
@@ -129,6 +130,7 @@ def _scaffold_item(ctx: Context, item_type, item_name):
         )
         config = loader.load(config_filepath.open())
         config.name = item_name
+        config.fingerprint = _compute_fingerprint(config_filepath.parent)
```

Comment:
Let's precompute it for scaffold packages.

Reply:
yes!